pulp.spec

```diff
@@ -458,13 +458,13 @@ Pulp provides replication, access, and accounting for software repositories.
 # If we are upgrading
 if [ $1 -gt 1 ] ; then
     %if %{pulp_systemd} == 1
-        /bin/systemctl stop pulp_workers > /dev/null 2>&1
-        /bin/systemctl stop pulp_celerybeat > /dev/null 2>&1
-        /bin/systemctl stop pulp_resource_manager > /dev/null 2>&1
+        /bin/systemctl stop pulp_workers > /dev/null 2>&1 || :
```

Comment:
Hmm. On the one hand, I'd like to know if our services become unstoppable, but on the other hand, having a scriptlet fail is a big no-no. I'm sure others will weigh in, but I think I'm okay with leaving them.